module/Application/test/Controller/IndexControllerTest.php

```diff
@@ -39,7 +39,7 @@ public function testIndexActionCanBeAccessed(): void
     public function testIndexActionViewModelTemplateRenderedWithinLayout(): void
     {
         $this->dispatch('/', 'GET');
-        $this->assertQuery('.container .jumbotron');
+        $this->assertQuery('.col .card');
```

Comment:
Actually, 

What the test is doing (and I corrected in my last commit.

It is checking that .container (which is a class from the layout) and .card (which is a class rendered in the /index/index.phtml file are both rendered.

Your suggested change will only test that the layout is rendered. This test is intended to test that both the layout.phtml file is rendered and that the actions view file is rendered within it. Which is why the assertQuery() needs to be passed a class from each file.

Reply:
I updated the original comment as well to clarify :)

Reply:
> This test is intended to test that both the layout.phtml file is rendered and that the actions view file is rendered within it.

And this can be done independently of the frontend framework:

- `html h1`
- `html h2`
- `body h1`
- …

Reply:
Absolutely. Its now:

body h1